### PR TITLE
Polkadot Protocol > On-Chain Governance > Origins & Tracks

### PR DESCRIPTION
**Pull request by kapetan3sid**
Created Open Gov Origins & Tracks page

### CODE REVIEW

File: polkadot-protocol/onchain-governance/overview.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ In OpenGov, origins and tracks are central to managing proposals and votes.
 
 Developers must be aware that referendums from different origins and tracks will take varying amounts of time to reach approval and enactment. The [Polkadot Technical Fellowship](https://wiki.polkadot.network/docs/learn-polkadot-technical-fellowship){target=\_blank} has the option to shorten this timeline by whitelisting a proposal and allowing it to be enacted through the [Whitelist Caller](https://wiki.polkadot.network/docs/learn-polkadot-opengov-origins#whitelisted-caller){target=\_blank} origin.
 
-Visit [Origins and Tracks Info](https://wiki.polkadot.network/docs/learn-polkadot-opengov-origins#origins-and-tracks-info){target=\_blank} for details on current origins and tracks, associated terminology, and parameters.
+Visit [Origins and Tracks Info](/){target=\_blank} for details on current origins and tracks, associated terminology, and parameters.
```
Comment (nhussein11) [inline polkadot-protocol/onchain-governance/overview.md:47]: @eshaben Sorry, I'm struggling to understand this change... what would be the reason for "(/)" here?

File: polkadot-protocol/onchain-governance/origins-tracks.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+For instance, a runtime upgrade (requiring a [`set_code`](https://paritytech.github.io/polkadot-sdk/master/frame_system/pallet/dispatchables/fn.set_code.html){target=\_blank} call) has far-reaching implications for the ecosystem compared to the approval of a treasury tip (via a [`report_awesome`](https://paritytech.github.io/polkadot-sdk/master/pallet_tips/pallet/dispatchables/fn.report_awesome.html){target=\_blank} call). As a result, distinct `Origins` are assigned for these actions, each with predefined parameters such as deposits, support, approval thresholds, and minimum enactment periods tailored to their specific requirements.
+
+!!!note
+    More information about terminology and parameters can be found on the [OpenGov Origins Wiki](https://wiki.polkadot.network/docs/learn-polkadot-opengov-origins){target=\_blanck} page.
+
+In the table below you can find all `Origins` and its `Tracks` on the Polkadot chain:
+
+| **ID** | **Origin**               | Max Deciding | Decision Deposit | Prepare Period | Decision Period | Confirm Period | Min Enactment Period | Min Approval      | Min Support       |
```
Comment (nhussein11) [inline polkadot-protocol/onchain-governance/origins-tracks.md:35]: @eshaben @dawnkelly09 How do you both feel about this table? I think we made an effort to replace tables in the past or something like that, but not sure

IMO, I don't see all the information in this table as necessary for devs, so maybe we can restructure this a bit and just leave fields like `Decision Deposit`, `Min Enactment Period`, and `Min Approval`; for the rest, we can add a link to these sections of the Polkadot wiki: https://wiki.polkadot.network/docs/learn-polkadot-opengov-origins#root

WDYT?

File: polkadot-protocol/onchain-governance/overview.md (hunk 1 above)
Comment (eshaben) [reply polkadot-protocol/onchain-governance/overview.md:47]: Mehhhh I must have somehow deleted it. It was supposed to change to be /polkadot-protocol/onchain-governance/origins-tracks/

File: polkadot-protocol/onchain-governance/origins-tracks.md (hunk 2 above)
Comment (eshaben) [reply polkadot-protocol/onchain-governance/origins-tracks.md:35]: Yeah, I think @dawnkelly09 mentioned something about cleaning that up based on Radha's feedback on a similar PR. So she's going to take a look at this and remove some of that info when she gets some time

File: polkadot-protocol/onchain-governance/overview.md (hunk 1 above)
Comment (eshaben) [reply polkadot-protocol/onchain-governance/overview.md:47]: I figured we should link to our page internally, but if we're not going to document the parameters and such, maybe we need two separate links here. One to link to our docs and one for the wiki

File: polkadot-protocol/onchain-governance/origins-tracks.md (hunk 2 above)
Comment (kapetan3sid) [reply polkadot-protocol/onchain-governance/origins-tracks.md:35]: @nhussein11 I'll remove `### Examples of Origins in OpenGov`, but content-wise, do you think page contains enough information about Origins and Tracks? (I am talking purely about the Learn section) 

@dawnkelly09 please don't butcher it too much :')

File: polkadot-protocol/onchain-governance/overview.md (hunk 1 above)
Comment (kapetan3sid) [reply polkadot-protocol/onchain-governance/overview.md:47]: yep @eshaben @nhussein11 this should be Wiki page - our page is high level overview, slimmed down without all the info about paramiters

**Review by dawnkelly09 — COMMENTED** [review]
Cut this back a ton and linked out to wiki for details.

File: polkadot-protocol/onchain-governance/origins-tracks.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+Tracks define a referendum's journey from submission to enactment, tailoring governance parameters to the impact of proposed changes. Each track operates independently and includes several key stages:
+
+- **Preparation** - time for community discussion before voting begins
+- **Voting** - period for token holders to cast their votes
+- **Decision** - finalization of results and determination of the proposal's outcome
+- **Confirmation** - period to verify sustained community support before enactment
+- **Enactment** - final waiting period before the proposal takes effect
+
+Tracks customize these stages with parameters like decision deposit requirements, voting durations, and approval thresholds, ensuring proposals from each origin receive the required scrutiny and process. For example, a runtime upgrade in the `Root Origin` track will have longer timeframes and stricter thresholds than a treasury request in the `Small Spender` track.
```
Comment (eshaben) [inline polkadot-protocol/onchain-governance/origins-tracks.md:38]:
```suggestion
Tracks customize these stages with parameters like decision deposit requirements, voting durations, and approval thresholds, ensuring proposals from each origin receive the required scrutiny and process. For example, a runtime upgrade in the Root Origin track will have longer timeframes and stricter thresholds than a treasury request in the Small Spender track.
```

File: polkadot-protocol/onchain-governance/origins-tracks.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Additional Resources
+
+- Visit [Origins and Tracks Info](https://wiki.polkadot.network/docs/learn-polkadot-opengov-origins#origins-and-tracks-info){target=\_blank} for a list of origins and tracks for Polkadot and Kusama including associated parameters.
+
+- See [Approval and Support](https://wiki.polkadot.network/docs/learn-polkadot-opengov#approval-and-support){target=\_blank} for a deeper dive into the approval and support system.
```
Comment (eshaben) [inline polkadot-protocol/onchain-governance/origins-tracks.md:44]:
```suggestion
- Visit [Origins and Tracks Info](https://wiki.polkadot.network/docs/learn-polkadot-opengov-origins#origins-and-tracks-info){target=\_blank} for a list of origins and tracks for Polkadot and Kusama including associated parameters

- See [Approval and Support](https://wiki.polkadot.network/docs/learn-polkadot-opengov#approval-and-support){target=\_blank} for a deeper dive into the approval and support system
```

**Review by nhussein11 — APPROVED** [review]
LGTM! 🚀

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**